src/Microsoft.ServiceFabric.Diagnostics/Metrics/Implementation/MeterBase.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.ServiceFabric.Diagnostics.Metrics.Implementation
+{
+    class MeterBase
```

Comment:
Let's make this class abstract and drop the term Base. https://en.wikipedia.org/wiki/All_your_base_are_belong_to_us :)